osstats_linux.go

```diff
@@ -14,9 +13,19 @@ import (
 
 var (
 	loadAvgTypes = []int{1, 5, 15}
-	nCPU         = float64(runtime.NumCPU())
+	nCPU float64
 )
 
+// init obtains nCPU using `proc.Stat()`, because `runtime.NumCPU()` doesn't
```

Comment:
It's a little odd to document `init`. (Like documenting `main`.) Move this comment one line down and change it to something like

```
	// Use proc to determine the number of CPUs.
	// This may be different from what runtime.NumCPU gives because
	// the gost process may have a restricted affinity mask.
	stat, err := proc.Stat()
```